src/Tokenizer/Token.php

```diff
@@ -67,6 +67,17 @@ public function __construct($token)
         }
     }
 
+    public function __debugInfo()
+    {
+        return array(
+            "\0+\0content" => $this->content,
```

Comment:
```php
<?php
class A
{
    private $content;
}

class B
{
    private $content;
    public function __debugInfo()
    {
        return array(
            "\0+\0content" => $this->content,
            "-content" => $this->content,
            "+content" => $this->content,
        );
    }
}

var_dump(new A(), new B());
die;
```

```bash
$ php /home/possum/work/symfony/test.php

/home/possum/work/symfony/test.php:20:
class A#1 (1) {
  private $content =>
  NULL
}
/home/possum/work/symfony/test.php:20:
class B#2 (3) {
  private $content =>
  NULL
  public $-content =>
  NULL
  public $+content =>
  NULL
}


```

(note the first `private` in the vardump of second object).

Something similar is done in the [caster](https://github.com/symfony/var-dumper/blob/master/Caster/Caster.php#L35) of SF VarDumper. I can't find any PHP docs as ref. as I think the output format is not formalized.